x/gamm/keeper/migrate.go

```diff
@@ -354,3 +354,33 @@ func (k Keeper) UpdateMigrationRecords(ctx sdk.Context, records []types.Balancer
 	}
 	return nil
 }
+
+func (k Keeper) LinkBalancerPoolWithClPoolRecords(ctx sdk.Context, records []types.BalancerToConcentratedPoolLink) error {
+	// TODO: check case where balancePoolExist but ClPool doesnot exist
```

Comment:
wanted to get thought on this 

- i think the frontend team just wants to link a CL pool to GAMM pool considering same denom. would this be necessary then?